src/crypto/tls/common.go

```diff
@@ -364,6 +365,15 @@ type ConnectionState struct {
 	// This means the client has offered ECH or sent GREASE ECH.
 	ECHOffered bool
 
+	// PeerTLSFlags is the set of TLS Flags sent by the Peer.
+	PeerTLSFlags []TLSFlag
+
+	// AgreedTLSFlags is the set of TLS Flags mutually supported by the Client and Server
```

Comment:
Also, wrap comments at 80 columns (here and below)
```suggestion
	// AgreedTLSFlags is the set of TLS Flags mutually supported by the Client and Server.
```